db/migrations.go

```diff
@@ -9,34 +9,71 @@ import (
 )
 
 type localMigration struct {
-	Name  string
-	Query string
+	Name  	  string
+	Condition []string
+	Query	  []string
```

Comment:
I added a Condition string array and turned Query in to a string array to support skipping a migration if it's not necessary, and to support multiple Condition and Query statements.